components/raftstore-v2/src/operation/command/admin/merge/prepare.rs

```diff
@@ -420,6 +442,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             pending_peers,
             req: Some(mem::take(req)),
         });
+        self.send_check_trim_status(store_ctx);
```

Comment:
Before this change, if send availability req returns error, it will not change merge status. But after this change, it will just ignore the error and enter WaitForTrimStatus. Is this expected?